roman.py

```diff
@@ -33,21 +33,21 @@ def get_roman_digit(number, base):
     if quotient >= 5:
         # do something including the half base
         if quotient == 9:
-            sys.stdout.write('%s%s' % (roman_prefix[(base*10)], roman_digit[(base*10)])) 
+            sys.stdout.write('%s%s' % (ROMAN_PREFIX[(base*10)], ROMAN_DIGIT[(base*10)])) 
```

Comment:
So the reason why I did `sys.stdout.write` is because I wanted to print each character as the script goes along.  `print`'s new line character got in the way. 

Perhaps a better way to do this is to use a string and keep concatenating each character, and print once the whole number is converted.